tests/testing_data.py

```diff
@@ -12,10 +12,10 @@ def __init__(self, **kw_args):
         intents = ["BookFlight", "GetWeather"]
         entities = [
             [
-                {"entity": "LocationFrom", "start": 22, "end": 26},
-                {"entity": "LocationTo", "start": 31, "end": 37},
+                {"entity": "LocationFrom", "start": 22, "end": 27},
```

Comment:
Hier wäre auch die Frage, ob die Änderung Auswirkungen auf andere Teile des Codes hat. In diesem Fall ist es ja implizit eine Änderung des Formats der Entities in NLUdataset, das sollte z.B. beim LUIS Vendor auch Änderungen nach sich ziehen, wenn ich mich recht erinnere...

Reply:
Tests laufen bei mir durch